extension/src/popup/components/manageAssets/AddAsset/index.tsx

```diff
@@ -80,7 +80,7 @@ export const AddAsset = () => {
   return (
     <Formik initialValues={initialValues} onSubmit={handleSubmit}>
       {({ dirty, errors, isSubmitting, isValid, touched }) => (
-        <Form>
+        <Form className="FormContainer">
```

Comment:
can you namespace this is as `AddAsset__FormContainer` to prevent possible collision? I could see .FormContainer being reused

Reply:
yup sure thing, done in e0b38d8e779eb7766a64537240d3247c2a53ae55